### PR TITLE
CLI problems with mod_suexec

### PR DESCRIPTION
**Pull request by rallek**
| Q                 | A
| ----------------- | ---
| Bug fix?          | no
| New feature?      | no
| BC breaks?        | no
| Deprecations?     | no
| Tests pass?       | yes
| Fixed tickets     | -
| Refs tickets      | -
| License           | MIT
| Doc PR            | -
| Changelog updated | [no]

Added a Comment for the case of mod_suexec

### CODE REVIEW

File: INSTALL-1.4.0.md [hunk 1]
```diff
@@ -111,6 +111,8 @@ $ php app/console zikula:install:finish
 ```
 Open your browser and login!
 
+Attention: If you have ``mod_suexec`` installed at Apache the CLI will run into permission problems. If you are not shure see phpinfo. mod_suexec often is used at shared hosting. In this case you have to use the Web Installer.
+
```
Comment (craigh) [inline INSTALL-1.4.0.md]: "sure" not "shure"

*have to* use the web installer? no choice? @Guite @cmfcmf

Comment (rallek) [inline INSTALL-1.4.0.md]: the CLI will create folders with the wrong ownership and wrong permissions.

**Comment by cmfcmf** [comment]
I am not sure if this really is the problem of `mod_suexec`.

**Comment by craigh** [comment]
> I think that's always the case when running from CLI and the cli user differs from the webserver's user.

hmm. **that** could be a problem, no? I wonder if that can be corrected?

**Comment by rallek** [comment]
mod_suexec was the idea of @shefik.

**Comment by cmfcmf** [comment]
See http://symfony.com/doc/current/book/installation.html#checking-symfony-application-configuration-and-setup. Symfony has the exact same problem. They also provide a CLI installer.

**Comment by craigh** [comment]
@rallek please add this link to the documentation also.

**Comment by craigh** [comment]
I guess the "good" news here is that Drupal (and other highly used CMS products that are migrating to or currently are based on Symfony) will all have the same problem :confused:

**Comment by craigh** [comment]
merging this but plan to reformat the link.

**Pull request merged**